src/cmds/mod.rs

```diff
@@ -442,18 +455,29 @@ pub enum SkfCommands {
     Update(UpdateFilters),
 }
 
+#[derive(Debug, Subcommand)]
+pub enum NetIdCommands {
+    List(ListNetIds),
+}
+
 #[derive(Debug, Subcommand)]
 pub enum OrgCommands {
     /// Get all Orgs
     List(ListOrgs),
     /// Get an Organization you own
     Get(GetOrg),
+    /// Create a new NetId
+    CreateNetId(CreateNetId),
```

Comment:
Should these be Admin Commands?

Reply:
anyone can call it since its just a solana instruction. The constraints on chain will prevent only the authority of the routing manager for actually creating it.